.gitignore

```diff
@@ -0,0 +1,24 @@
+*.class
+*.log
+*.iml
+.idea/*
+
+# sbt specific
+.cache/
+.history/
+.lib/
+dist/*
+project/project/
+project/target/
+src/main/java/
+src/main/resources/
+src/main/scala-2.10/
+src/test/java/
+# src/test/resources/
```

Comment:
we could remove this, as we do not want to ignore our test resources

Reply:
corrected 👍